src/Repository/AbstractRepository.php

```diff
@@ -344,35 +346,56 @@ public function getEntitiesFromTarget(EntityInterface $entity, EntityInterface $
     }
 
     /**
-     * @param array $filters
-     * @return mixed
+     * Returns entities that contain the $fieldValue of the $fieldName.
+     * The match is performed using the LIKE comparison operator.
+     * The result set is paginated.
+     *
+     * @param string $fieldName
+     * @param string $fieldValue
+     * @param int $from
+     * @param int $size
+     * @return array
      */
-    public function getNumberOfEntities(array $filters)
+    public function getEntitiesByField(string $fieldName = 'id', string $fieldValue ="", int $from = 0, int $size = 999999999999999)
     {
-        $query = "SELECT count(*) as entitiesNumber FROM " .
-            $this->getTableName() .
-            $this->getFilters($filters);
-        $dbStmt = $this->pdo->prepare($query);
-        foreach ($filters as $fieldName => &$value) {
-            $dbStmt->bindParam(':' . $fieldName, $value);
+        $query = 'SELECT count(*) as entitiesNumber FROM ' . $this->getTableName();
```

Comment:
does this ever happen? it seems to me that it doesn't cause you always bind limit&offset